File: 2019/01/1.py
import math
fuel1 = fuel2 = 0
with open('input.txt', 'rb') as textfile:
    for line in textfile:
        fuel1 += math.floor(float(line) / 3) - 2
        while math.floor(float(line) / 3) - 2 > 0:
            fuel2 += math.floor(float(line) / 3) - 2
            line = math.floor(float(line) / 3) - 2
print(fuel1, fuel2)

File: 2019/01/1.jl
using DelimitedFiles

input = readdlm("input.txt", Int)

function p2(n)
    fuel = 0
	while n÷3 -2 > 0
        fuel += n÷3 -2
        n = n÷3 -2
	end
	return fuel
end

println(sum(input .÷ 3 .- 2))
println(sum(p2.(input)))
